tools/generate_uuids.py

```diff
@@ -4,10 +4,12 @@
 
 parser = argparse.ArgumentParser(description="Generate UUIDs for info*.json files that don't already have them.")
 parser.add_argument("directory", help="the directory to search for info*.json files")
+parser.add_argument("-v", "--verbose", action="store_true", help="List all the files for which uuid's were generated")
 parser.add_argument("-n", "--new", action="store_true", help="generate new UUIDs for all files, even if they already have one")
 args = parser.parse_args()
 
-skip_dirs = [".git"]
+# Added elements in the skip directory because elements should not contain a uuid
```

Comment:
Don't comment on what was changed. Comments should reflect the current status of the file. It's also a bit redundant to say that we are skipping `elements` because they shouldn't contain UUIDs. I suggest simply deleting this comment.